docs/api/features/get.asciidoc

```diff
@@ -1,29 +1,28 @@
 [[features-api-get]]
-=== Get Features
-
-experimental[This API is *experimental* and may be changed or removed completely in a future release.]
+=== Get features API
+++++
+<titleabbrev>Get features</titleabbrev>
+++++
 
 Retrieves all {kib} features.
 
-Note: You cannot access this endpoint via the Console in {kib}.
```

Comment:
Any reason we're removing this note? I know we get confused users on the discuss forums pretty frequently because they're trying to run these in the Console app.

Reply:
Yes! For 7.5, I'm going to create a `Getting started` piece for the APIs that will cover how to use the Kibana APIs. This should remove the confusion around using (or not using) our Kibana endpoints in the Console. 

But I see where this might cause confusion in 7.4. I stuck the note on the first page of the REST API page since the note applies to this whole section, but if you think it would be better to keep this note on all of the pages in the meantime, I am happy to add it back. 

Let me know!

Reply:
I think the problem with putting it only on the top-level page for this section is that often times we'll link users directly to a specific API, so they may not see this section.

The ideal solution here is to make the Console app actually work with Kibana APIs as well so we don't have to have this in the first place. I agree that having this note littered throughout the docs is not great.

I'll leave this decision up to you. I think if we see an uptick in confused users we can change it in the future.

Reply:
Deal. I'll make the getting started piece for the APIs my main priority for 7.5. 

Thank you for all of your input!